google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -760,3 +762,8 @@ def export_images(
             index += 1
 
         return output_filenames
+
+    def to_hocr(self,filename: str) -> str:
+        hocr = _Hocr(documentai_pages=self.pages, documentai_text=self.text,filename=filename)
+        
+        return hocr.export_hocr()
```

Comment:
To be clear, I am not requesting to change this at this point, let's continue the discussion.

Reply:
I've tried to use only helper functions without the loading object steps and the result is a super slow export, with this implementation it takes milliseconds for the export to return a string and the second implementation it takes seconds up to a minute.

Reply:
So I can try to add this logic inside the normal wrappers to see if there is any affect I believe there will be. I believe the reason it would take more time is because we're adding another operation to each token, line , paragraph, block and page wrapping process. So when you have N pages with N tokens,lines,paragraphs and blocks the time added to each wrapping I believe would be a lot(I think the time complexity would be O(n^5). As opposed to only exporting the objects to hOCR when the user actually wants to do export the wrapped document.